test/tests_bin.cpp

```diff
@@ -21,63 +21,100 @@ extern int32_t i32;
 extern ThingSetArrayInfo int32_array;
 extern ThingSetArrayInfo float32_array;
 
-extern ThingSetArrayInfo pub_serial_array;
-
 extern ThingSetBytesBuffer bytes_buf;
 
-void test_bin_get_output_ids()
+void test_bin_get_meas_ids_values()
 {
-    uint8_t req[] = { TS_GET, 0x18, ID_OUTPUT, 0xF7 };
+    uint8_t req[] = { TS_GET, ID_MEAS };
 
     uint8_t resp[100];
     ts.process(req, sizeof(req), resp, sizeof(resp));
 
     char resp_hex[] =
-        "85 83 "     // successful response: array with 3 elements
+        "85 A3 "     // successful response: map with 3 elements
         "18 71 "
+        "FA 41 61 99 9A "        // 14.1
         "18 72 "
-        "18 73 ";
+        "FA 40 A4 28 F6 "        // 5.13
+        "18 73 "
+        "16";
 
     uint8_t resp_expected[100];
     int len = hex2bin(resp_hex, resp_expected, sizeof(resp_expected));
 
     TEST_ASSERT_EQUAL_HEX8_ARRAY(resp_expected, resp, len);
 }
 
-void test_bin_get_output_names()
+void test_bin_get_meas_names_values()
 {
-    uint8_t req[] = { TS_GET, 0x18, ID_OUTPUT, 0x80 };
+    uint8_t req[] = { TS_GET, 0x64, 0x6D, 0x65, 0x61, 0x73 };
 
     uint8_t resp[100];
     ts.process(req, sizeof(req), resp, sizeof(resp));
 
     char resp_hex[] =
-        "85 83 "     // successful response: array with 3 elements
+        "85 A3 "     // successful response: map with 3 elements
         "65 42 61 74 5F 56 "
+        "FA 41 61 99 9A "        // 14.1
         "65 42 61 74 5F 41 "
-        "6C 41 6D 62 69 65 6E 74 5F 64 65 67 43";
+        "FA 40 A4 28 F6 "        // 5.13
+        "6C 41 6D 62 69 65 6E 74 5F 64 65 67 43 "
+        "16";
+
+    uint8_t resp_expected[100];
+    int len = hex2bin(resp_hex, resp_expected, sizeof(resp_expected));
+
+    TEST_ASSERT_EQUAL_HEX8_ARRAY(resp_expected, resp, len);
+}
+
+void test_bin_get_single_value()
+{
+    uint8_t req[] = { TS_GET, 0x18, 0x71 };
```

Comment:
constify - also all other uses of constant data in testing.

Reply:
Thanks, valid point. Will do that during rebase after #10 is merged.